.eslintrc.js

```diff
@@ -9,6 +9,7 @@
         "*.ts"
       ],
       "parserOptions": {
+        tsconfigRootDir: __dirname,
```

Comment:
All other properties have quotes so this could also have them (`"tsconfigRootDir"`)

Reply:
Ideally none of the property keys would have quotes in a js file (this is a fragment of changing the file from json to js) but I'll just add the quotes for now